.travis.yml

```diff
@@ -140,7 +141,7 @@ jobs:
 
     # Run the tests against without compilation (GNU/Linux, Python 3.5)
     - stage: lint and pure python test
-      <<: *stage_linux_no_compile
+      <<: *stage_linux
```

Comment:
I don't understand this change.

Reply:
This is a quick hack by Diego to fix Travis for Linux.

Reply:
It was mostly for installing the `apt` dependencies in the second job at the first "Lint and pure python test" stage - aimed indeed for making the minimal change that helped moving the PR forward. Revising the travis configuration would be welcome (as Jay mentioned, #1620 would be a good time!).